File: 1295. Find Numbers with Even Number of Digits/solution.java
class Solution {
  public int findNumbers(int[] nums) {
    int count = 0;
    for (int index = 0; index < nums.length; index++){
      int num = nums[index];
      if ((num > 9 && num < 100) || (num > 999 && num < 10000) || num == 100000){
        count += 1;
      }
    }

    return count;
  }
}

File: 1295. Find Numbers with Even Number of Digits/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var findNumbers = function(nums) {
  let count = 0;
  for (let num of nums){
    if ((num > 9 && num < 100) || (num > 999 && num < 10000) || num === 100000){
      count += 1;
    }
  }

  return count;
};